src/main.js

```diff
@@ -1 +1,4 @@
-// Write your code here
\ No newline at end of file
+console.log(4+5);
```

Comment:
კოდი ასე უფრო წაკითხვადია!
```suggestion
console.log(4 + 5);
```